src/algo/api_nfdomains.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+import requests
+from settings_csv import ALGO_NFDOMAINS
+
+
+# API documentation: https://editor.swagger.io/?url=https://api.testnet.nf.domains/info/openapi3.yaml
+class NFDomainsAPI:
+    session = requests.Session()
+
+    def get_address(self, name):
+        endpoint = f"nfd/{name}"

```

Comment:
Whoever wrote this, thank you!
Is there a way to have multiple addresses feed into staketax?
An NFD can technically link to many addresses - it looks like you grab the owner in [line #18](https://github.com/hodgerpodger/staketaxcsv/pull/167/files).  The owner is obviously super important, but when users 'link' addresses, the first is what we'll call the 'deposit' address.  Basically, what assets go to if someone sends to an .algo address.  This would be caAlgo[0] if found.  

If staketax can do multiple account reporting, then you can grab:
owner, caAlgo(*), and nfdAccount.  The nfdAccount is the account address of the NFD itself and its vault.  Later on, users will be able to use them as delegated vaults.

Please don't hesitate to reach out to our #dev-talk channel on Discord with any questions.  
Love this project btw - it was a life-saver for my taxes recently since Koinly is pretty broken for ASAs.

Cheers!

Reply:
FYI: The mainnet endpoint will be https://api.nf.domains

Reply:
Thanks for taking a look Patrick!

For the moment staketax can only process a single address, but if an NFD can map to multiple accounts it should be possible to produce a zip file with multiple reports rather than a single CSV.

As per your suggestion for the time being I'll use the deposit address if found (and owner as a fallback).

Also I'll switch to the mainnet endpoint once it goes live in a few days :)